### PR TITLE
Add setting to show each class's armor on separate lines in vault

### LINKED ISSUE #10355: allow the armor groupings to also be separated onto different lines like the weapon groupings
### Proposed change

request from discord [here](https://discord.com/channels/316217202766512130/1232101793228525578)  

probably should have a separate toggle for this since that toggle only appears when grouping weapons and armor groupings are not optional. 

image

### How does this fit into your workflow?

just a visual consistency that would be nice.

### PR DESCRIPTION
**Pull request by FlaminSarge**
Closes #10355 

Screenshot 2024-08-08 at 1 50 45 AM
Screenshot 2024-08-08 at 1 53 08 AM
Screenshot 2024-08-08 at 1 49 51 AM

### CODE REVIEW

**Comment by FlaminSarge** [comment]
I think the text on the setting doesn't read well, would like suggestions on it.

**Comment by bhollis** [comment]
You don't need to block on API changes - you can add new settings directly to `initial-settings.ts` and we move them over occasionally.

**Comment by FlaminSarge** [comment]
> You don't need to block on API changes - you can add new settings directly to `initial-settings.ts` and we move them over occasionally.

Oh cool, that's a nice setup.

File: config/i18n.json [hunk 1]
```diff
@@ -1193,7 +1193,8 @@
     "ReverseSort": "Toggle forward/reverse sort",
     "SetSort": "Sort items by:",
     "SetVaultWeaponGrouping": "Group vault weapons by:",
-    "VaultWeaponGroupingStyle": "Separate groups on different lines",
+    "VaultArmorGroupingStyle": "Separate each class's armor on different lines",
```
Comment (bhollis) [inline config/i18n.json:1196]: Maybe this?

```suggestion
    "VaultArmorGroupingStyle": "Separate armor on different lines by class",
```

**Review by bhollis — APPROVED** [review]

**Pull request merged**